### PR TITLE
Performance improvement

### PR DESCRIPTION
**Pull request by tomasdev**
Leverage virtual list to significantly improve performance of the page, both on load and on scroll.

### CODE REVIEW

File: android_fonts/vue-virtual-scroll-list.js [hunk 1]
```diff
@@ -0,0 +1,602 @@
+// taken from https://github.com/tangbc/vue-virtual-scroll-list/blob/master/dist/index.js
```
Comment (rsheeter) [inline android_fonts/vue-virtual-scroll-list.js:1]: we don't have to do something about license, put it into a /third_party dir or something?

Comment (tomasdev) [reply android_fonts/vue-virtual-scroll-list.js:1]: https://github.com/tangbc/vue-virtual-scroll-list/blob/master/LICENSE MIT license.
I think for now it's okay to have it laying around. We'll re-do this page soon enough.

Comment (rsheeter) [reply android_fonts/vue-virtual-scroll-list.js:1]: Lets at least shove it into a third_party dir and reference the license, similar to https://github.com/googlefonts/noto-emoji/tree/master/third_party

Comment (rsheeter) [reply android_fonts/vue-virtual-scroll-list.js:1]: Should be fine to do post-merge.

**Pull request merged**